R/api_url.R

```diff
@@ -109,10 +109,10 @@ api_url <- function(
   # End of validation
 
   endpoint_base <- list(
-    dev = "https://dev.statistics.api.education.gov.uk/api/",
-    test = "https://test.statistics.api.education.gov.uk/api/",
-    preprod = "https://pre-production.statistics.api.education.gov.uk/api/",
-    prod = "https://statistics.api.education.gov.uk/api/"
+    dev = "https://dev.statistics.api.education.gov.uk/",
+    test = "https://test.statistics.api.education.gov.uk/",
+    preprod = "https://pre-production.statistics.api.education.gov.uk/",
+    prod = "https://statistics.api.education.gov.uk/"
```

Comment:
Apologies if I've missed it elsewhere in here, what prompted this change?

Reply:
Dunc and Nick? The old url with the extra "api/" doesn't exist anymore.

Reply:
Ah fair! Probably information I knew and should have retained better than I did... Makes sense really, as otherwise API was in the URL twice